src/Backend/Modules/Pages/Engine/Model.php

```diff
@@ -840,11 +840,11 @@ public static function getSubtree(array $navigation, int $parentId): string
 
                 // insert link
                 $html .= '    <a href="' . BackendModel::createUrlForAction(
-                    'Edit',
-                    null,
-                    null,
-                    ['id' => $page['page_id']]
-                ) . '"><ins>&#160;</ins>' . htmlspecialchars($page['navigation_title']) . '</a>' . "\n";
+                        'Edit',
```

Comment:
hmm now that I see this, all this extra white space is also not oke

Reply:
I have changed the original model.php from the master, so I think it's ok now?

Reply:
Yes